.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ]  Based on the requirements `test.md` file should be ignored by git. So kindly add the `test.md` into .gitignore file not to be shown in the GitHub repository.